src/content/en/updates/2019/09/fresher-sw.md

```diff
@@ -93,6 +104,40 @@ if ('serviceWorker' in navigator) {
 }
 ```
 
+## Checks for updates to imported scripts
+
+Prior to Chrome 78, any service worker script loaded via
+[`importScripts()`](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts)
+would be retrieved only once (checking first against the HTTP cache, or via the
+network, depending on the `updateViaCache` configuration). After that initial
+retrieval, it would be cached internally by the browser, and never re-fetched.
```

Comment:
It's potentially worth saying "stored" instead of "cached" to avoid confusion with the HTTP cache. (up to you) It would also be refetched if the main script changed and the importScript wasn't in the HTTP cache (but that may be too detailed)

Reply:
Changed.